constants.yml

```diff
@@ -1547,3 +1547,6 @@ partial_data_available:
     'full-data-available-from-department-of-the-economy' : "Refer to the Department of the Economy for further information about this company"
     'full-data-available-from-the-company' : "Contact the company directly for further information"
     'full-data-available-from-financial-conduct-authority-mutuals-public-register' : "Refer to the Financial Conduct Authority Mutuals Public Register for further information about this company"
+corporate_annotation_type:
+    'confirmed-intelligence-of-unusual-behaviour-from-the-fca' : "Confirmed Intelligence of unusual behaviour from the FCA"
```

Comment:
Is it `Confirmed Intelligence...` or `Confirmed intelligence...`?

Reply:
Will find out about this one also 👍